database/schema/upgrade_scripts/051-inisghts_rule-additional-data.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE insights_rule ADD description_text TEXT;
+ALTER TABLE insights_rule ADD summary_text TEXT;
+ALTER TABLE insights_rule ADD generic_text TEXT;
+ALTER TABLE insights_rule ADD reboot_required BOOLEAN;
+ALTER TABLE insights_rule ADD playbook_count INT;
+ALTER TABLE insights_rule ADD change_risk INT;
+ALTER TABLE insights_rule ADD kbase_node_id INT;
```

Comment:
we need the `impacted_systems_count` or is it stored in our DB?

Reply:
That's good point, maybe we can take this information from advisor instead of counting manually, I am only afraid of some caching issues with that and inconsistencies between apps.